hypha/apply/funds/tables.py

```diff
@@ -204,6 +204,10 @@ def get_screening_statuses(request):
         id__in=ApplicationSubmission.objects.all().values('screening_statuses__id').distinct('screening_statuses__id'))
 
 
+def get_categories(request):
+    return Category.objects.filter(filter_on_dashboard=True)
```

Comment:
This should return all categories and the options belong to each category. I do not think categories themselves should have a checkbox, only the options.